src/config.py

```diff
@@ -489,7 +493,8 @@ def client_properties(self) -> List[str]:
         client_properties = [
             f'sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required username="{username}" password="{password}";',
             "sasl.mechanism=SCRAM-SHA-512",
-            f"security.protocol={self.security_protocol}",  # FIXME: will need changing once multiple listener auth schemes
+            f"security.protocol={self.security_protocol}",
+            # FIXME: security.protocol will need changing once multiple listener auth schemes
```

Comment:
**nitpick:** Maybe keep this on one line?

In the utterly unimportant case where we need to remove lines from `client_properties`, it's nice to have only 1 line per item to remove/comment-out.

Reply:
I believe the linter automatically did this, since I did not change these lines myself, but just run the usual `tox -e ...` commands

Reply:
All good then!